apiary.apib

```diff
@@ -1082,6 +1122,15 @@ Any response that does not provide a ```200 OK``` response code should be treate
 + `keyCode`: yes (string, required) - The string representation
 + `valueCode`: Yes Sir (string, required) - The display representation of the type code
 
+## Business Asset Type Group (Base Audit Object)
```

Comment:
There is a new standard in place for how base audits are handled, but since Core wasn't apart of that clean up story its okay to be consistent to this project for now.  A story has been created to attack Core in the future.